packages/react-native-codegen/e2e/__tests__/components/__snapshots__/GeneratePropsH-test.js.snap

```diff
@@ -321,7 +321,7 @@ namespace facebook::react {
 enum class EnumPropNativeComponentViewAlignment { Top, Center, BottomRight };
 
 static inline void fromRawValue(const PropsParserContext& context, const RawValue &value, EnumPropNativeComponentViewAlignment &result) {
-  auto string = (std::string)value;
+  auto string = std::string_view{(std::string)value};
```

Comment:
Casting to `string` and *then* to `string_view` doesn't really make sense, I'm afraid. You're creating a temporary `string` (incurring the same copy as before btw), then effectively creating a dangling pointer to its contents. See [proof by godbolt + asan](https://godbolt.org/z/f56zsGo5s).

I don't have enough context, but it might make sense to create a non-copying `const string&` / `string_value` accessor *on `RawValue`*, if we know the string is stored in some stable location that is safe to access for the life of our `value` reference.

Reply:
TIL I learned about compiler explorer, pretty awesome! 
OK, I'll keep this PR up a bit to explore an alternative path, but noted the current version is _not_ what we want.